remoulade/middleware/logging_metadata.py

```diff
@@ -0,0 +1,29 @@
+from ..logging import get_logger
+from .middleware import Middleware
+
+
+class LoggingMetadata(Middleware):
+    def __init__(self):
+        self.logger = get_logger(__name__, type(self))
+
+    @property
+    def actor_options(self):
+        return {
+            "logging_metadata",
+            "logging_metadata_getter",
+        }
+
+    def update_options_before_create_message(self, options):
+        logging_metadata = options.get("logging_metadata")
+        logging_metadata_getter = options.get("logging_metadata_getter")
```

Comment:
maybe use pop ?

Reply:
I can't use pop since I'm now using get_option to get this option's value